src/java/fr/paris/lutece/portal/service/database/LuteceTransactionManager.java

```diff
@@ -108,13 +109,16 @@ private Plugin getPlugin( )
     {
         if ( _plugin == null )
         {
-            if ( StringUtils.isNotBlank( _strPluginName ) )
+            synchronized ( LuteceTransactionManager.class )

```

Comment:
Several thread can see `_plugin` as null, and then proceed to initialize it. The null check should be inside the `synchronized` block.
Furthermore, this is an instance method. Why is the synchronisation on the class level ?

Reply:
Will be corrected